### ISSUE TITLE
Any kind of documentation for using this with Contao?

### ISSUE THREAD

**Issue by philvdb** [contao-bootstrap/grid#64]
I've been developing Contao for 14 years now and just needed to do some basic column work for a pro bono project. I usually don't do much frontend but thought I should be able to figure this out.

I was not.

I've been searching the web up and down and there is documentation on the bootstrap grid, but I have found none about how to use this extension in Contao. I've "defined" a grid in the Theme settings and used it for wrapping grid content elements - tried placing some "separators" in between but the output it generates makes little sense to me and does not conform to bootstrap's documentation examples.

I am obviously missing something since a number of people seem to have gotten this to work - good on them - but *any* kind of list of necessary steps to get this thing to work *in Contao* would have been great.

Usually, I would peruse the source code until I get a handle on what a developer was thinking or - more likely - I will develop my own extension because it's sometimes faster than trying to figure someone else's undocumented stuff out. But like I said, this was supposed to be a pro bono project and my time for it has run out. It's going to remain ugly and live in eternity without columns.

**Comment by Kahmoon** [comment]
Did you see the contao-bootstrap section in the contao forum?
https://community.contao.org/de/forumdisplay.php?169-contao-bootstrap

**Comment by philvdb** [comment]
Thanks for your quick reply and I had not realized that forum was dealing with the same extension.

Still can't find any thread that would just tell you the steps to use this extension. Like these are the three things you have to do.

- Define grid in Theme
- Add content element wrappers
- Add separators

**Comment by Kahmoon** [comment]
You first have to install/upload bootstrap. This module **won´t** download any bootstrap on installation.

**Comment by philvdb** [comment]
Whoa, you're saying there are external files (the bootstrap framework?) I'd have to install myself and there is **no mention of that** in the README.md file - not even under the heading "Install"?

Call me crazy and I can imagine to everyone "in the know" this will have been obvious. But hey - I've seen there have even been calls for funding for this project - if you want that, I would strongly encourage you to make the project *accessible* so more people are able to *use* it.

@Kahmoon thank you so much for your efforts with me here but it's really a shame you're doing it in an issue like this. This should be a PR for the repo's README.md file!

**Comment by Kahmoon** [comment]
Yes, you have to install those files itself. So you are flexible with versions, which would not be the case if this module would do this for you.

The module just outputs contao elements with bootstrap classes.

**Comment by philvdb** [comment]
> But basically this is a topic for the contao forum. Its not an issue.

I beg to differ here. This blatant lack of even the most basic accessibility makes it unnecessarily hard for new users to use the extension. And I do not agree the solution is to have a thread on some forum explaining the basic installation.

It *should* ideally be part of the repo, like pretty much any other open source software.

> Yes, you have to install those files itself. So you are flexible with versions and plugins, which would not be the case if this module would do this for you.

That makes sense and it is good to know. But it makes no sense this information would not be in the README.md.
